cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/embankment.json

```diff
@@ -49,5 +25,29 @@
     "elevation-units": "m",
     "bounding-office-id": "LRL",
     "nearest-city": "Davis"
+  },
+  "structure-type": {
+    "active": true,
+    "office-id": "CWMS",
+    "tooltip": "TOOLTIP_STRUCT",
+    "display-value": "STRUCT"
+  },
+  "upstream-side-slope": 15.0,
+  "downstream-side-slope": 90.0,
+  "structure-length": 25.0,
+  "height-max": 5.0,
```

Comment:
Why is it height-max and not max-height?

Reply:
huh, looks like every other usage of height+max in the database is `max_height`. I'll update